.github/CODEOWNERS

```diff
@@ -1,5 +1 @@
-* @hashicorp/consul-core
-
-# release configuration
-/.release/                              @hashicorp/release-engineering
-/.github/workflows/build.yml            @hashicorp/release-engineering
```

Comment:
Does RelEng want to maintain their review requirement for these?

Reply:
Asked and awaiting response :)